README.md

```diff
@@ -15,8 +15,16 @@ We want to pull politics into the 21st century, and we're starting from the grou
 ## How can I help?
 
 * First, join us on [Slack](http://goo.gl/forms/8SJRDlo7Lx2rUsan1).
-* Then, take a look at [our todo list](http://waffle.io/getmovement/national-voter-file), and see where you'd like to contribute.
-* Finally, learn how to [view and modify the relational model itself](https://github.com/getmovement/national-voter-file/tree/master/docker), the core of our work.
+* Take a look at [our todo list](http://waffle.io/getmovement/national-voter-file).
+* Learn about [the code](https://github.com/getmovement/national-voter-file/tree/master/docker) itself!
+
+Here's the bulk of the work:
+
+1. [Collect voter files for every state](https://trello.com/b/IlZkwYc0/national-voter-file-states-pipeline) and store them securely.
+2. Write state-specific Python scripts to turn them into a standardized input file (SIF).
```

Comment:
Is there somewhere we can dive into what an SIF is more? Just link out somewhere?